### PR TITLE
Remove hardcoded extensions in browserify constructor's options

### PR DESCRIPTION
**Pull request by liitfr**

### CODE REVIEW

File: lib/index.coffee [hunk 1]
```diff
@@ -44,7 +44,7 @@ module.exports = (opts) ->
 
       @b = browserify
         entries: @files
-        extensions: ['.js', '.json', '.coffee']
+        extensions: opts.opts.extensions
```
Comment (jescalan) [inline lib/index.coffee]: This is cool, but can you also default them as they exist so this is not enormously breaking?

```coffee
opts.opts.extensions or ['.js', '.json', '.coffee']
```

Comment (liitfr) [inline lib/index.coffee]: Hello !
Is this thing not already done in line 16 ?

Comment (jescalan) [inline lib/index.coffee]: Ah it totally is hah! Good call. Not sure how that mixup happened.

**Comment by jescalan** [comment]
Sorry for the delay and thanks for your contribution!

**Pull request merged**